packages/angular/cli/README.md

```diff
@@ -178,12 +181,34 @@ You can also use `ng new foo --link-cli` to automatically link the `@angular/cli
 Please read the official [npm-link documentation](https://docs.npmjs.com/cli/link)
 and the [npm-link cheatsheet](http://browsenpm.org/help#linkinganynpmpackagelocally) for more information.
 
-To run the Angular CLI test suite use the `node tests/run_e2e.js` command.
-It can also receive a filename to only run that test (e.g. `node tests/run_e2e.js tests/e2e/tests/build/dev-build.ts`).
+To run the Angular CLI test suite use the `node ./tests/legacy-cli/run_e2e` command.
+It can also receive a filename to only run that test (e.g. `node ./tests/legacy-cli/run_e2e tests/legacy-cli/e2e/tests/build/dev-build.ts`).
 
 As part of the test procedure, all packages will be built and linked.
 You will need to re-run `npm link` to re-link the development Angular CLI environment after tests finish.
 
+### Debugging with VS Code
```

Comment:
This configuration must be launched from the CLI repo, right? I'm not too sure what situation is it useful in. One doesn't usually run any `ng` commands inside the repo proper.

Reply:
I used to do this in angular-cli before the devkit merge, in order to debug issues repo steps and get to the point more quickly. As you can see, the configuration use another project directory as "cwd". I didn't found how to do this well with npm link. I think we may need to add some script in order to link every built project and add a watch mode, but this would make this PR much more larger that it was intended to be. This section could go to another PR with this kind of script.

Reply:
Ah, I had not noticed the `cwd` to another project!